### DISCUSSION TITLE
How to open the ganache homescreen to begin creating a workspace

### DISCUSSION THREAD

**Discussion by richwealthyandsuccessful** [ConsenSys-archive/ganache#3251] [Q&A]
Hi,

I'm using a HP Chromebook with Chrome OS & Linux Beta. 

I've managed to download Ganache v7.3.0 (@ganache/cli: 0.4.0, @ganache/core: 0.4.0) but when I run the command in the terminal.

$ ganache

It starts running ganache from the terminal showing the accounts, private keys, wallet etc, but I'm not able to get the ganache home screen to appear to begin creating a workspace.

I've also tried to open the file outside of the terminal by going to: files > my files > linux files > node modules > ganache

But this doesn't give me an option to open ganache outside of the terminal either?

**Comment by apdan510** [comment]
Rich,

You will need Ganache with the User interface to see the dashboard. If you have a linux OS running please try installing Ganache UI with [ganache-2.5.4-linux-x86_64.AppImage](https://github.com/trufflesuite/ganache-ui/releases/download/v2.5.4/ganache-2.5.4-linux-x86_64.AppImage)

There might be some compatibility issues with your Chromebook. Please give it a try and let me know.

Kind regards,
Alex | Truffle Support

**Comment by richwealthyandsuccessful** [comment]
Hi Alex, 

Sorry for the delay, I attempted the first method you recommended but to no avail.

Since then I've followed the installation advice from the links below in an attempt to use Truffle for VS Code:

https://trufflesuite.com/blog/build-on-web3-with-truffle-vs-code-extension/
https://trufflesuite.com/docs/vscode-ext/installation-guide/

But again, I'm not able to get Truffle for VSCode to open and the command palette is not showing, despite me downloading the dependency requirements.

Any ideas? I understand there maybe compatibility issues, but I'm still trying my best to proceed.

Thanks

Marcus
